examples/resources/consul_certificate_authority/resource.tf

```diff
@@ -2,7 +2,7 @@
 resource "consul_certificate_authority" "connect" {
   connect_provider = "consul"
 
-  config_json = jsondecode({
+  config_json = jsonencode({
```

Comment:
This appeared to be a typo from #341 , but please double-check me

Reply:
Yeah seems like an autocomplete mixup